rego/snyk.rego

```diff
@@ -14,13 +14,23 @@
 
 package snyk
 
+__physical_or_logical_id(resource) = ret {
+	is_string(resource.attributes.id)
+	not resource.attributes.id == ""
+	ret := resource.attributes.id
+} else = ret {
+	ret := resource.id
+}
+
 resources(resource_type) = ret {
 	ret := [obj |
 		resource := input.resources[resource_type][_]
+		# id := object.get(resource.attributes, "id", resource.id)
```

Comment:
comment can be deleted now that we have the `__physical_or_logical_id` function

Reply:
Thanks! Fixed in 28fef08